src/murmur/Meta.cpp

```diff
@@ -228,6 +229,14 @@ void MetaParams::read(QString fname) {
 						break;
 				}
 			}
+
+			++nif;
+		}
+
+		if (nif == 0) {
+			qWarning("Meta: Unable to query network interfaces. Assuming support for both IPv4 and IPv6.");
```

Comment:
I don't like the "query" here. If we were able to check individual interfaces for their "up" and "running" and "loopback" states, we surely were able to query them!?

`QNetworkInterface::allInterfaces()` [can return an empty list on a "failure"](http://doc.qt.io/qt-5/qnetworkinterface.html#allInterfaces).

I think we should make it two checks; if the list is empty, we were not able to "query" them - because of some kind of failure. If the list is non-empty but we do not find a interface matching our filter (up, running, no loopback) we warn about that. That would provide more specific and elaborative messages.

Looking the current changes against the change description - I described two "error" cases we handle as one here. So our change is not only for the described "could not query"/"interfaces not ready yet" (whatever that means), but we now also do this on `allInterfaces()`s failure - which we ignore before (at least here).

I guess configuration determines the interfaces gettable through `allInterfaces` - not sure at what point (after/on the systemd network interfaces target?), then they get enabled/upped, once the finished "upping" they are "running". If we will always get the interfaces, but they are just not running or enabled yet, we can separate the failure case.